install/kubernetes/cilium/charts/config/templates/configmap.yaml

```diff
@@ -268,9 +268,14 @@ data:
 {{- end }}
 
   masquerade: {{ .Values.global.masquerade | quote }}
+  enable-bpf-masquerade: {{ .Values.global.bpfMasquerade | quote }}
 {{- if .Values.global.egressMasqueradeInterfaces }}
   egress-masquerade-interfaces: {{ .Values.global.egressMasqueradeInterfaces }}
 {{- end }}
+{{- if and .Values.global.ipMasqAgent .Values.global.ipMasqAgent.enabled }}
```

Comment:
This requires a warning somewhere in the docs or in helm generation, the `ip-masq-agent` config map needs to be set in the same namespace where Cilium is running.

Reply:
Good point, I will mention it in the docs.